core-web/libs/edit-content/src/lib/components/dot-edit-content-field/dot-edit-content-field.component.spec.ts

```diff
@@ -50,6 +70,7 @@ describe('FIELD_TYPES and FIELDS_MOCK', () => {
 describe.each([...FIELDS_MOCK])('DotEditContentFieldComponent all fields', (fieldMock) => {
     let spectator: Spectator<DotEditContentFieldComponent>;
```

Comment:
I wonder if we should create a parent describe that handles the initialization of the component, in both describes we do almost the same, maybe we can merge this process into one describe and then do the `describe.each` inside

Reply:
Is it worthy it?

Reply:
I don't really know, maybe we can left it as it is, not sure if this component will need more tests that the ones it has